src/test/java/de/tum/in/www1/artemis/localvcci/LocalVCLocalCITestService.java

```diff
@@ -364,7 +364,7 @@ public String commitFile(Path localRepositoryFolder, Git localGit, String fileNa
         Path testFilePath = localRepositoryFolder.resolve(fileName);
         Files.createFile(testFilePath);
         localGit.add().addFilepattern(".").call();
-        RevCommit commit = localGit.commit().setMessage("Add " + fileName).call();
+        RevCommit commit = localGit.commit().setSign(false).setMessage("Add " + fileName).call();
```

Comment:
The newly extracted function isn't called in here as well.

Reply:
Yes, because neither the GitService nor GitUtilService was used in this file, I wanted to avoid introducing one or another because I was unsure which one to use.

Reply:
I added the GitService as a dependency, now the method is used